src/MyConnection.h

```diff
@@ -42,6 +42,16 @@ class MyConnection : boost::noncopyable {
       mysql_options(pConn_, MYSQL_READ_DEFAULT_FILE,
                     Rcpp::as<std::string>(default_file).c_str());
 
+    if (!ssl_ca.isNull())
+      mysql_ssl_set(
```

Comment:
Shouldn't this function be called also if ssl_ca is NULL (in this case with a NULL argument)?